pyscf/md/integrators.py

```diff
@@ -494,3 +495,104 @@ def _next_velocity(self, next_accel):
         necessary equations of motion for the velocity is
             v(t_i+1) = v(t_i) + 0.5(a(t_i+1) + a(t_i))'''
         return self.veloc + 0.5 * self.dt * (self.accel + next_accel)
+        
+        
+class NVTBerendson(_Integrator):
+    '''Berendsen (constant N, V, T) molecular dynamics
+
+    Args:
+        method : lib.GradScanner or rhf.GradientsMixin instance, or
+        has nuc_grad_method method.
+            Method by which to compute the energy gradients and energies
+            in order to propagate the equations of motion. Realistically,
+            it can be any callable object such that it returns the energy
+            and potential energy gradient when given a mol.
+            
+        T      : float
+            Target temperature for the NVT Ensemble. Given in K.
+            
+        taut   : float
+            Time constant for Berendsen temperature coupling. Given in atomic units.
+
+    Attributes:
+        accel : ndarray
+            Current acceleration for the simulation. Values are given
+            in atomic units (Bohr/a.u.^2). Dimensions is (natm, 3) such as
+
+             [[x1, y1, z1],
+             [x2, y2, z2],
+             [x3, y3, z3]]
+    '''
+
+    def __init__(self, method, **kwargs):
```

Comment:
I think you need `__init__` to take `T` and `taut`. This function should look more like
```python
def __init__(self, method, T, taut, **kwargs):
    self.T = T
    self.taut = taut
    self.accel = None
    super().__init__(method, **kwargs)
```

Reply:
I have added that! But should be redundant I guess!!